config/posix_secondary.toml

```diff
@@ -0,0 +1,18 @@
+[uptane]
+ecu_hardware_id = "local-fake-secondary"
+
+[storage]
+type = "sqlite"
+path = "storage"
+sqldb_path = "secondary.db"
+
+[network]
+port = 9050
+primary_ip = "127.0.0.1"
+primary_port = 9040
+
+[pacman]
+type = "fake"
+
+[logger]
+loglevel = 0
```

Comment:
Will anything be using this config, or is it just an example? Is it possible to have meta-updater use these?

Reply:
Nothing uses this conf at the moment, it's rather intended for an user/developer to run a fake secondary, so they can use it as it is or modify it according to their needs. If no such conf provided in the repo then a developer will need to figure out a fake secondary config by himself/herself which is additional burden, IMHO.